test/core/bin.js

```diff
@@ -132,7 +132,12 @@ _.each(fs.readdirSync(fixtureLoc), function (binName) {
       };
 
       var optionsLoc = testLoc + "/options.json"
-      if (fs.existsSync(optionsLoc)) _.merge(opts, require(optionsLoc));
```

Comment:
@sebmck notice that the test runner assumed all options could be represented as JSON (which is true, before this PR). I didn't know whether to convert all `options.json` to be a JS module or just use something like this?